core/src/main/scala/kanaloa/reactive/dispatcher/queue/Autothrottler.scala

```diff
@@ -103,56 +101,90 @@ trait Autothrottler extends Actor with ActorLogging with MessageScheduler {
   }
 
   private def optimize(currentSize: PoolSize): ScaleTo = {
+    val newSize = Autothrottler.optimize(currentSize, perfLog, settings)
+    ScaleTo(newSize, Some("optimizing"))
```

Comment:
smallish comment:  This `reason` field seems to have minimal value, other than being asserted on in some specs and printed one time in a log.debug message.  Maybe consider removing it at some point.